### PR TITLE
code review project

### PR DESCRIPTION
**Pull request by Bantealem**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script src="js/index.js"></script>
+</body>
+
+</html>
```
Comment (Bantealem) [inline index.html:113]: Please remove all the free lines in the index.html.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+.submit-button {
+    display: block;
+    width: 100%;
+    padding: 0.75rem;
+    background: var(--color-green);
+    color: inherit;
+    border-radius: 2px;
+    cursor: pointer;
+}
```
Comment (Bantealem) [inline style.css:159]: Please remove all the free lines in style.css

**Review by Bantealem — COMMENTED** [review]
Changes required ♻️
please check my comments below and make all required changes.
Your pull request should a descriptive title and a well detailed description of the action you've done in you branch, and this will help the reviewer to understand the summary of the activities that you've worked on.
You should have .gitignore file that hold node_modules folders. This will stop those folders to be pushed to remote. check this for [more](https://stackoverflow.com/questions/27850222/what-is-gitignore/27850270#27850270)
You should have a readme file. This is important to your project reader.

**Comment by Nemwel-Boniface** [comment]
## Hello @Bantealem      :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  Thank you for making the changes that were requested by the previous reviewer. :clap:



### Great review!
   - ✔Comments are polite suggestions rather than orders.
   - ✔Good explanation as to why the changes are requested.
   - ✔Used the Github review function correctly.


Your project is complete! Congratulations! 🎉 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**